app/experimenter/experiments/migrations/0034_auto_20190319_2119.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.1.7 on 2019-03-19 21:19
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("experiments", "0033_auto_20190319_1820")]
+
+    operations = [
+        migrations.AddField(
+            model_name="experiment",
+            name="normandy_id",
+            field=models.PositiveIntegerField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name="experiment",
+            name="client_matching",
+            field=models.TextField(
+                blank=True,
+                default="Prefs:\n\nStudies:\n\nAny additional filters:\n    ",
```

Comment:
This doesn't look right. Did this get left out of a previous PR?

Reply:
It suuuuurrreeeee did.  https://github.com/mozilla/experimenter/pull/1068

Reply:
Maybe we should add a step to continuous integration to check that no migrations got missed.

Reply:
Filed #1079

Reply:
I think we should, yeah. We [did it on Normandy](https://github.com/mozilla/normandy/pull/1591) recently, and it was a pretty easy check.

Do you want to leave this here and merge as is, or do you want to do something to fix this?

Reply:
Well It has to go in and I don't see the point of splitting it into its own tiny PR so I'm fine with this as is.